SU-SWS/decanter
# Tweaking modular-spacing mixin to be more user friendly

# READY FOR REVIEW

# Summary
- Tweaking the modular-spacing mixin so each individual property doesn't get wrapped into a separate set of media queries.
- It now follows the shorthand notation of padding/margin, e.g., `@include modular-spacing('margin', 3 null 2 -1);` will generate output for each breakpoint:
```
{
margin-top: calculate-modular-spacing(3); 
margin-bottom: calculate-modular-spacing(2);
margin-left: calculate-modular-spacing(-1);
}
```

# Needed By (Date)
- Whenever

# Urgency
- Not urgent

# Steps to Test

1. Pull this branch
1. Test on a component using e.g., `@include modular-spacing('margin', 0 -4 3 null);`
2. Run `grunt styleguide` and see if it produces the expected result.

# Affected Projects or Products
- Decanter
- event calendar

# Associated Issues and/or People
- @scott's event calendar
